.github/workflows/ci.yml

```diff
@@ -75,66 +45,20 @@ jobs:
         uses: actions/setup-python@v4.3.0
         with:
           python-version: ${{ env.python_version }}
-      - name: Install system dependencies
-        if: matrix.platform == 'ubuntu-22.04'
-        run: |
-          sudo apt-get install -y flatpak flatpak-builder
-      - name: Get packages
-        uses: actions/download-artifact@v3
-        with:
-          name: packages
-          path: dist
       - name: Install packages
         run: pip install briefcase
-      - name: Create App
-        run: |
-          cd tests/apps
-          cat verify-${{ matrix.framework }}.config | briefcase new --template .
-      - name: Build App
-        run: |
-          cd tests/apps/verify-${{ matrix.framework }}
-          briefcase create
-          briefcase build
-          briefcase package --adhoc-sign
       - name: Build Xcode project
         if: matrix.os_name == 'macOS'
         run: |
           cd tests/apps/verify-${{ matrix.framework }}
           briefcase create ${{ matrix.os_name }} Xcode
+          echo 'CREATE COMPLETE'
```

Comment:
I'm guessing this (and the BUILD comment below) are stray debug?

Reply:
Yep, I thought I had removed those!

Reply:
Turns out I had forgotten to commit the change.